src/olympia/devhub/templates/devhub/addons/submit/done.html

```diff
@@ -19,7 +19,7 @@ <h3>{{ _("Version Submitted for Review") }}</h3>
       {{ _("You’re done! This version will be available after it passes review. "
            "You will be notified when it is reviewed.") }}
     </p>
-  {% elif uploaded_version.is_webextension and not addon.auto_approval_disabled %}
+  {% elif uploaded_version.is_ready_for_auto_approval() %}
```

Comment:
I made a mistake when I suggested this change in the issue, `is_ready_for_auto_approval` is a `property`, not a method, so you should remove the `()` to make this work.